### PR TITLE
[packet action] Clarify packet actions: COPY, TRAP, and LOG

### PR DESCRIPTION
**Pull request by yxieca**

### CODE REVIEW

**Comment by yxieca** [comment]
retest this please

File: inc/saiswitch.h [hunk 1]
```diff
@@ -95,18 +95,43 @@ typedef enum _sai_packet_action_t
...
 
-    /** This is a combination of SAI packet action COPY and FORWARD. */
+#define SAI_PACKET_ACTION_BIFURCATE SAI_PACKET_ACTION_LOG
+
+    /**
+     * @brief Packet action bifurcate
+     *
+     * This is a combination of SAI packet action COPY and FORWARD:
+     * A copy of the original packet is sent to CPU port, the original
+     * packet is forcefully forwarded in the pipeline.
```
Comment (atitjain) [inline inc/saiswitch.h]: What does the **forcefully forwarded** mean here. Does it mean any action drop is cancelled?

File: inc/saiswitch.h [hunk 2]
```diff
@@ -95,18 +95,43 @@ typedef enum _sai_packet_action_t
...
+     * @brief Packet action trap
+     *
+     * This is a combination of SAI packet action COPY and DROP:
+     * A copy of the original packet is sent to CPU port, the original
+     * packet is forcefully dropped from the pipeline.
+     */
     SAI_PACKET_ACTION_TRAP,
 
-    /** This is a combination of SAI packet action COPY and FORWARD. */
+#define SAI_PACKET_ACTION_BIFURCATE SAI_PACKET_ACTION_LOG
```
Comment (kcudnik) [inline inc/saiswitch.h]: please remove this define, and fix comments errors

File: inc/saiswitch.h (hunk 1 above)
Comment (yxieca) [reply inc/saiswitch.h]: Your understanding is correct. I'll update PR when after collecting more feedback.

File: inc/saiswitch.h (hunk 2 above)
Comment (yxieca) [reply inc/saiswitch.h]: We might be able to remove this definition, I heard suggestions of moving away from LOG action.

Cam you elaborate on 'fix comments errors'?

Comment (kcudnik) [reply inc/saiswitch.h]: why not be able to remove like you just added this ? and comments errors - click red X in this page to see jenkins console logs

Comment (yxieca) [reply inc/saiswitch.h]: Thanks! :-)

**Pull request merged**